hw/bsp/frdm-k82f/src/arch/cortex_m4/startup_MK82F25615.S

```diff
@@ -272,7 +272,7 @@ __isr_vector:
     .long 0xFFFFFFFF
     .long 0xFFFFFFFF
     .long 0xFFFFFFFF
-    .long 0xFFFFFFFE
+    .long 0xFFFF3DFE
```

Comment:
If you are setting `BOOTSRC_SEL` to internal flash shouldn't `BOOTPIN_OPT` be 1 to select `BOOTSRC_SEL`, in other words `0xFFFF3FFE`?

Reply:
Yes, you are right, I misread the doc. That is what I intended to do.

Reply:
Fixed.